R/rrmake.R

```diff
@@ -5,8 +5,9 @@ make_find_root_file <- function(criterion) {
   }))
 }
 
-make_fix_root_file <- function(criterion, path) {
+make_fix_root_file <- function(criterion, path, subdir) {
```

Comment:
`subdir = NULL` looks best to me, changing the test may mean breaking downstream code.

Reply:
I can set `subdir=NULL` as default for `make_fix_root_file()`. 

Are there any other functions that rely on `make_fix_root_file()` whose behaviors might be altered?

I'd like to set up a unit test, but the formatting of the current unit tests is a little bit beyond my understanding